doc/man7/EVP_PKEY-DSA.pod

```diff
@@ -31,21 +82,150 @@ The private key value.
 
 =item "p" (B<OSSL_PKEY_PARAM_FFC_P>) <unsigned integer>
 
-A DSA or Diffie-Hellman "p" value.
+A DSA or Diffie-Hellman prime "p" value.
 
 =item "q" (B<OSSL_PKEY_PARAM_FFC_Q>) <unsigned integer>
 
-A DSA or Diffie-Hellman "q" value.
+A DSA or Diffie-Hellman prime "q" value.
 
 =item "g" (B<OSSL_PKEY_PARAM_FFC_G>) <unsigned integer>
 
-A DSA or Diffie-Hellman "g" value.
+A DSA or Diffie-Hellman generator "g" value.
+
+=item "seed" (B<OSSL_PKEY_PARAM_FFC_SEED>) <octet string>
+
+An optional domain parameter I<seed> value used during generation and validation
+of I<p>, I<q> and canonical I<g>.
+For validation this needs to set the I<seed> that was produced during generation.
+
+=item "gindex" (B<OSSL_PKEY_PARAM_FFC_GINDEX>) <integer>
+
+Sets the index to use for canonical generation and verification of the generator
+I<g>.
+Set this to a positive value from 0..FF to use this mode. This I<gindex> can
+then be reused during key validation to verify the value of I<g>. If this value
+is not set or -1 then unverifiable generation of generator I<g> will be used.
```

Comment:
is not set or **_is_** -1 then unverifiable generation of **_the_** generator

Reply:
Fixed